src/Test/TestCombatMods.java

```diff
@@ -162,7 +162,7 @@ private boolean testRetribution()
     // Clean up
     testMap.removeUnit(infA);
     testMap.removeUnit(infB);
-    performEvents(testGame, venge.army.initTurn(testMap));
+    turn(testGame, 2);
```

Comment:
Huh. I wonder why we even do another turn at the end of this test. Also why it would require two turn changes instead of just one.

Reply:
The power won't end if the turn doesn't end.
2 is to get back to the turn we started on.